hack/forks.data

```diff
@@ -0,0 +1,13 @@
+https://github.com/openshift/origin master vendor/k8s.io/kubernetes https:/github.com/openshift/kubernetes release-1.7.0
+https://github.com/openshift/origin master vendor/k8s.io/kubernetes/staging/sr/k8s.io/code-generator https://github.com/openshift/kubernetes-code-generator master
+https://github.com/openshift/kubernetes release-1.7.0 staging/src/k8s.io/apimachinery https://github.com/openshift/kubernetes-apimachinery release-1.7.0
+https://github.com/openshift/kubernetes release-1.7.0 staging/src/k8s.io/apiserver https://github.com/openshift/kubernetes-apiserver release-1.7.0
+https://github.com/openshift/kubernetes release-1.7.0 staging/src/k8s.io/client-go https://github.com/openshift/kubernetes-client-go release-1.7.0
+https://github.com/openshift/kubernetes release-1.7.0 staging/src/k8s.io/metrics https://github.com/openshift/kubernetes-metrics release-1.7.0
+https://github.com/openshift/origin master vendor/github.com/docker/distribution https://github.com/openshift/docker-distribution release-2.6.2
+https://github.com/openshift/origin master vendor/github.com/emicklei/go-restful-swagger12 https://github.com/openshift/emicklei-go-restful-swagger12 release-1.0.1
+https://github.com/openshift/origin master vendor/github.com/google/cadvisor https://github.com/openshift/google-cadvisor release-0.26.1
```

Comment:
only this one had patches.  Find a better indicator in the verify.  This is a pain.